clientv3/client.go

```diff
@@ -310,6 +310,10 @@ func (c *Client) getToken(ctx context.Context) error {
 		var resp *AuthenticateResponse
 		resp, err = auth.authenticate(ctx, c.Username, c.Password)
 		if err != nil {
+			if err == rpctypes.ErrAuthNotEnabled {
+				// return err if err == rpctypes.ErrAuthNotEnable ,Don't retry other endpoints
```

Comment:
I would suggest to just say something like "// return err without retrying other endpoints" (to keep it clean as the if condition is self explanatory)

Reply:
Sure,Thanks @spzala

Reply:
No problem @cfc4n and thank you for addressing the suggestions!